src/OpenGL/MonoMacGameView.cs

```diff
@@ -73,7 +74,14 @@ public MonoMacGameView (CGRect frame, NSOpenGLContext context) : base(frame)
 				NSOpenGLPixelFormatAttribute.ColorSize, 24,
 				NSOpenGLPixelFormatAttribute.DepthSize, 16 };
 
-			pixelFormat = new NSOpenGLPixelFormat (attribs);
+			try {
+				pixelFormat = new NSOpenGLPixelFormat (attribs);
+			} catch (Exception) {
```

Comment:
can you be more specific ? i.e. you're not handling `OutOfMemoryException` with that

Reply:
It doesn't look like we're getting a more specific exception when the issue happens: https://gist.github.com/VincentDondain/2fb0b74548a390a25a33b81ad2fa172e

```
Unhandled Exception:
System.Exception: Could not initialize an instance of the type 'AppKit.NSOpenGLPixelFormat': the native 'init*' method returned nil.
```

Reply:
hmm... that's something we should change - but not in the PR :)